From: =?utf-8?b?IllhbyBXZWkgKOmtj+mKmOW7tyki?= <mwei@lxde.org>
Date: Wed, 15 Jun 2016 19:06:27 +0800
Subject: Fix sparc build by removing lib64 path detection

 Not yet pushed to upstream
Last-Update: 2013-10-01
---
 configure | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/configure b/configure
index fc5d148..fde4214 100755
--- a/configure
+++ b/configure
@@ -205,11 +205,6 @@ fi
 
 
 LIB='lib'
-if [ -d /usr/lib64 -a -d /lib64 -a ! -d /lib/x86_64-linux-gnu ]; then
-LIB='lib64'
-# for fedora 64bit
-GTK_QUERY_IM_64='-64'
-fi
 
 QT_IM='N'
 QT_IM_DIR=/usr/$LIB/qt3/plugins/inputmethods
