From: =?utf-8?b?IkNoYW5nWmh1byBDaGVuICjpmbPmmIzlgKwpIg==?=
 <czchen@debian.org>
Date: Wed, 15 Jun 2016 19:06:28 +0800
Subject: Fix FTBFS on kfreebsd-*

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index 32ca4fb..3df832f 100755
--- a/configure
+++ b/configure
@@ -28,7 +28,7 @@ if [ $? = 0 ]; then
 fi
 
 FREEBSD=0
-uname | grep FreeBSD > /dev/null 2> /dev/null
+uname | grep FreeBSD | grep -v GNU/kFreeBSD > /dev/null 2> /dev/null
 if [ $? = 0 ]; then
   FREEBSD=1
 fi
