From: =?utf-8?b?IkNoYW5nWmh1byBDaGVuICjpmbPmmIzlgKwpIg==?=
 <czchen@debian.org>
Date: Thu, 30 Jun 2016 00:09:02 +0800
Subject: Reproducible build

---
 pin-juyin.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/pin-juyin.cpp b/pin-juyin.cpp
index e385d3a..385fdf5 100644
--- a/pin-juyin.cpp
+++ b/pin-juyin.cpp
@@ -20,6 +20,8 @@ int main()
   if ((fp=fopen(fnamein, "r"))==NULL)
     p_err("cannot open %s", fnamein);
 
+  memset(pinju, 0, sizeof(pinju));
+
   while (!feof(fp)) {
     char tt[128];
 
